nuxeo-services/nuxeo-permissions/src/test/java/org/nuxeo/ecm/permissions/ACLJsonEnricherTest.java

```diff
@@ -162,4 +165,36 @@ public void testExtendedFetchingAsRegularUser() throws Exception {
         }
     }
 
+    @Test
+    public void testCompatibility() throws Exception {
+        DocumentModel root = session.getDocument(new PathRef("/"));
+        JsonAssert json = jsonAssert(root, CtxBuilder.enrichDoc("acls").get());
+        json = json.has("contextParameters").isObject();
+        json.properties(1);
+        json = json.has("acls").length(1).has(0);
+        json.has("name").isEquals("local");
+        json.has("aces").isArray();
+        json.hasNot("ace");
+
+        try {
+            harness.deployContrib("org.nuxeo.ecm.permissions", "test-acl-enricher-compat-config.xml");
```

Comment:
As the `aces` field is already tested in another field, I've simplified the test ot test only the compatibility using a `@LocalDeploy` as suggested.